slides/pages/signup.ftd

```diff
@@ -0,0 +1,103 @@
+-- import: ui.fifthtry.com/slides/components/action-page
+-- import: ui.fifthtry.com/slides/site-data
+-- import: ui.fifthtry.com/i18n/en
+-- import: ui.fifthtry.com/i18n/hi
+
+-- string lang: en
+
+
+-- component page:
+string github-url:
+string login-url:
+
+-- action-page.page:
+site-data: site-data
+
+-- signup-page:
+site-data: site-data
+github-url: $page.github-url
+login-url: $page.login-url
+lang: en
+
+-- end: action-page.page
+
+-- end: page
+
+
+
+
+
+
+
+
+
+
+-- component signup-page:
+module site-data: site-data
+module lang: en
+string github-url:
+string login-url:
+
+-- ftd.row:
+width: fill-container
+background.solid: $inherited.colors.background.base
+
+-- ftd.image:
+src: $assets.files.slides.assets.background.png
+width.fixed.percent: 60
+fit: cover
```

Comment:
You have to figure out the common stuff from login.ftd and this file, and make a component based on that, and this file must be what is unique to this file.